### PR TITLE
[fix-doc]: correct documentation on maths library configuration

### PR DESCRIPTION
**Pull request by adoussot**
TODO: this should also be done with all remaining `enable_`configuration
(e.g: `enable_uml` which seems also deprecated)

### CODE REVIEW

File: samples/sample.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ Also, any indented block is considered a code block.  If `enable_highlight` is `
 
 ## Math
 
-When `enable_mathjax` is `true`, inline math can be included \\(\frac{\pi}{2}\\) $\pi$
+When specific library to display Maths in the browser (e.g: MathJax or KaTeX) are correctly set through `"js"`, `"css"` and `"markdown_extensions"` configuration field, inline math can be included \\(\frac{\pi}{2}\\) $\pi$
```
Comment (facelessuser) [inline samples/sample.md]: I would probably rephrase this to:

```
Math can be displayed in the browser using MathJax or Katex. The feature can be enabled by correctly configuring the `"js"`, `"css"`, and `"markdown_extensions"` configuration field. This allows for inline math to be included \\(\frac{\pi}{2}\\) $\pi$.
```

Comment (adoussot) [reply samples/sample.md]: thanks! see 6c34866
